src/octod/api/repos.d

```diff
@@ -78,6 +78,35 @@ struct Repository
     }
 
     /**
+        Fetches all repository branches
+
+        Returns:
+            array of tag structs for all GitHub branches in this repo
+     **/
+    Tag[] branches ( )
```

Comment:
Please rename `struct Tag` to `struct GitRef` and keep old name as deprecated alias if you want to reuse it for branches too.